### PR TITLE
#649 Move top level paths to x.D

### PR DESCRIPTION
**Pull request by MichaelStritt**
### Linked issue

Check out #649

### CODE REVIEW

File: Modules/SubModule_ASL/xASL_wrp_RegisterASL.m [hunk 1]
```diff
@@ -582,18 +582,18 @@
 %% Admin
 PathMaskTemplate = fullfile(x.SESSIONDIR, 'Mask_Template.nii');
 PathTemplate = fullfile(x.SESSIONDIR, 'Mean_CBF_Template.nii');
-[Fpath, Ffile] = xASL_fileparts(x.PathMask);
-x.PathMask2 = fullfile(Fpath, [Ffile '2.nii']);
+[Fpath, Ffile] = xASL_fileparts(x.D.PathMask);
+x.D.PathMask2 = fullfile(Fpath, [Ffile '2.nii']);
 x.PathCBF = fullfile(x.SESSIONDIR, 'LowRes_Mean_CBF_Template.nii');
```
Comment (jan-petr) [inline Modules/SubModule_ASL/xASL_wrp_RegisterASL.m:587]: why not upgrading also PathCBF?

**Review by jan-petr — CHANGES_REQUESTED** [review]
Looks all good - one small comment?

File: Modules/SubModule_ASL/xASL_wrp_RegisterASL.m (hunk 1 above)
Comment (MichaelStritt) [reply Modules/SubModule_ASL/xASL_wrp_RegisterASL.m]: Done 👍

**Review by jan-petr — APPROVED** [review]
OK

**Pull request merged**